webpack/build-utils.js

```diff
@@ -6,4 +8,9 @@ const minFilenamePart = isMin ? '.min' : '';
 
 const lightFilenamePart = isLight ? '.light' : '';
 
-module.exports = { isMin, isLight, minFilenamePart, lightFilenamePart };
+const getProfilesPathPattern = (outputPath) => ({
+  from: path.resolve(outputPath, '../src/profiles'),
+  to: path.resolve(outputPath, 'profiles')
+});
```

Comment:
why did you need this webpack thing? can’t package.json exports handle that?

Reply:
I mean by that we can have raw json files instead of js files that exports json objects